WiseBuy/APIs/APIHelper.h

```diff
@@ -0,0 +1,29 @@
+//
+//  APIHelper.h
+//  WiseBuy
+//
+//  Created by Mai Ngo on 7/21/22.
+//
+
+#import <Foundation/Foundation.h>
+#import "Item.h"
+#import "Deal.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol APIDelegate <NSObject>
+
++ (void)fetchDeals: (NSString *)barcode;
+
+@end
+
+@interface APIHelper : NSObject
+@property (weak, nonatomic) id<APIDelegate> delegate;
++ (Item *)createItem: (NSString *)name : (NSString *)description : (NSString *)barcode : (NSString *)imageUrl;
```

Comment:
nit: It's really difficult to read code when you don't add the param names in the method declaration. We usually add in a name for each parameter because it (1) makes the code more readable, (2) reduces the chance that you pass in a value in the wrong spot (e.g. switching description and barcode)

For something like this, we would typically structure it as:
+ (Item *)createItem:(NSString *)name description:(NSString *)description barcode:(NSString *)barcode imageURL:(NSString *)imageUrl;

Reply:
will fix this